src/content-tags-drawer/utils.js

```diff
@@ -1,2 +1,13 @@
 // eslint-disable-next-line import/prefer-default-export
 export const extractOrgFromContentId = (contentId) => contentId.split('+')[0].split(':')[1];
+export const debounce = (func, delay) => {
+  let timeoutId;
+
+  return (...args) => {
+    clearTimeout(timeoutId);
+
+    timeoutId = setTimeout(() => {
+      func.apply(this, args);
+    }, delay);
+  };
+};
```

Comment:
Instead of writing our own version you can just use the one from lodash since lodash is already in use in this app.

Reply:
Great idea, updated here: https://github.com/openedx/frontend-app-course-authoring/pull/737/commits/9789faf9585d122fbcb76b241d9f79ff917a24da